templates/app-scaffold/AI_RULES.md

```
@@ -168,6 +168,31 @@ loudly — apps are read-mostly by design. Don't try to work around this.
+  if (res.status === "needs_approval" && res.request_id) {
+    setRow({ state: "awaiting" }); // show "Awaiting approval"
+    const poll = window.setInterval(async () => {
+      const { state } = await getActionStatus(res.request_id!);
+      if (state === "approved" || state === "rejected") {
+        window.clearInterval(poll); // stop once resolved — never poll forever
+        setRow({ state }); // reflect approved/posted or rejected in the UI
+      }
+    }, 5_000);
+    // Also clear the interval on unmount (useEffect cleanup).
+  }
+  ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)